src/components/EnsureLoggedInContainer.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import {connect} from 'react-redux'
```

Comment:
We should really decide on the formatting for these imports, single quotes vs double quotes, (I vote single), and semi-colon or no semi-colon (I don't really care). I think this should get covered by the linter / git actions ticket you have though.

Reply:
Singe seems to be the standard way of doing it. No semi colon looks a little cleaner I think. But yea, the linter should take care of it